JeMPI_Apps/JeMPI_API_KC/src/main/resources/keycloak.json

```diff
@@ -2,6 +2,7 @@
   "realm" : "${env.KC_REALM_NAME}",
   "resource" : "${env.KC_JEMPI_CLIENT_ID}",
   "auth-server-url" : "${env.KC_API_URL}",
+  "frontend-kc-url": "${env.KC_FRONTEND_URL}",
```

Comment:
I don't see any docs on the keycloak webpage on this. What does it do?

Reply:
Oh I see, it's the reason for the custom verifier. Did yu try using 'proxy-url' as somehting more standard?

See - https://www.keycloak.org/docs/latest/securing_apps/#keycloak-java-adapters

Reply:
Hmm...hadnt looked at that. Could I mark this as  a **Potential improvement for later** (for now). Might take sometime to look into this, and update everything if need be

Reply:
Sure, happy with that.